README.md

```diff
@@ -6,7 +6,7 @@
 [![Zenodo Badge](https://zenodo.org/badge/18918/sympy/sympy.svg)](https://zenodo.org/badge/latestdoi/18918/sympy/sympy)
 [![codecov Badge](https://codecov.io/gh/sympy/sympy/branch/master/graph/badge.svg)](https://codecov.io/gh/sympy/sympy)
 
-A Python library for symbolic mathematics.
+![SymPy Banner](banner.svg)
 
 <https://sympy.org/>
```

Comment:
```suggestion
[![SymPy Banner](banner.svg)](https://sympy.org/)

```

Reply:
Should I open a new pr after performing the suggested changes or this will do?

Reply:
You can just commit the change in this PR. It was only necessary to open a new PR last time because there were some commits that added files and others that removed the same files and we don't want those kinds of commits in the history.